WeScan/Scan/CaptureSessionManager.swift

```diff
@@ -122,6 +128,18 @@ final class CaptureSessionManager: NSObject, AVCaptureVideoDataOutputSampleBuffe
     }
     
     internal func capturePhoto() {
+        
+        let captureConnection = photoOutput.connections.first { (connection) -> Bool in
+            return connection.inputPorts.first(where: { (port) -> Bool in
+                port.mediaType == .video
+            }) != nil
+        }
+        guard captureConnection != nil || captureConnection?.isEnabled == true || captureConnection?.isActive == true else {
+            let error = ImageScannerControllerError.capture
```

Comment:
Would be great to make this error a bit more descriptive, like `ImageScannerControllerError.captureConnectionFailed`

Reply:
As same as the first comment.

@AvdLee thanks for you comment.